modules/microshift-updating-rpms-z.adoc

```diff
@@ -6,7 +6,7 @@
 [id="microshift-applying-patch-updates-rpms_{context}"]
 = Applying patch updates using RPMs
 
-Updating {microshift-short} on non `rpm-ostree` systems such as {op-system-base-full} requires downloading then updating the RPMs. A system reboot is not required for patch updates. For example, use the following procedure to upgrade from 4.17.0 to 4.17.1.
+Updating {microshift-short} on non `rpm-ostree` systems such as {op-system-base-full} requires downloading then updating the RPMs. A system reboot is not required for patch updates. For example, use the following procedure to upgrade from 4.17.1 to 4.17.2.
```

Comment:
Isn't it a good practice to at least restart MicroShift (using systemctl restart) ?

Reply:
This is out of scope for this PR. In this PR only patch versions are updated from 4.17.0 to 4.17.1

Reply:
but still, @dhensel-rh  point is valid - the "system reboot is bnot required" is highly misleading, because a restart of MicroShift is not only good practice, but actually a must have to for the patch upgrade to take effect.
If this is not in scope of this PR, please open a new bug to get this fixed.

Reply:
ok, so I will remove the line "A system reboot is not required for patch updates" and add a line in procedure to restart microshift.